maker/src/connection.rs

```diff
@@ -39,13 +39,18 @@ use xtras::SendInterval;
 
 const TCP_TIMEOUT: Duration = Duration::from_secs(10);
 
-fn parse_vergen_version(vergen_string: &str) -> semver::Version {
-    let vergen = semver::Version::parse(vergen_string).expect("daemon version to parse");
+fn parse_vergen_version(vergen_string: &str) -> Result<semver::Version> {
+    let vergen = semver::Version::parse(vergen_string)
+        .context("Could not parse daemon_version: {vergen_string}")?;
```

Comment:
Do you know what the version was when it paniced?

Reply:
unfortunately, it printed only the offending character, which was, if I remember correctly `<` (None that this would not have happened in production, as we only make weird tags where we append git hashes when building from `master`.